Makefile

```diff
@@ -24,6 +26,15 @@ build-tools:
 	cd third_party/yosys && PREFIX=${INSTALL_DIR} $(MAKE) -j`nproc` && PREFIX=${INSTALL_DIR} $(MAKE) install
 	# Build Yosys plugins
 	cd third_party/yosys-plugins && export PATH=${INSTALL_DIR}/bin:${PATH} && $(MAKE)
+	# Build nextpnr
+	cd third_party/nextpnr && cmake -DARCH=xilinx . && make -j`nproc`; \
+		cp nextpnr-xilinx ${INSTALL_DIR}/bin; \
+		NEXTPNR_DIR=${INSTALL_DIR}/share/nextpnr; \
+		mkdir -p $$NEXTPNR_DIR; \
```

Comment:
It seems a bit weird that you have to run these commands for nextpnr?

Reply:
I didn't notice I could use the INSTALL PREFIX here as well, removed the `cp` of the binary.

For what regards the other files (`bba` and `bin`) they need to be generated somewhere, and there should be one for each supported device. I think they are the equivalent of the `rr_graph`. Anyways, their generation could be moved in some other make target maybe, but they definetly need to be build before tests run.